src/utilities/validateReport.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Check if the given parameter is a valid Report based
+ * on its data structure and type and return a boolean.
+ *
+ * @param   {object}  parsedData  The data to check.
+ * @param   {string}  type        Type of Report to check.
+ */
+
+import Ajv from 'ajv';
+
+// Expected json structure for Table reports
+const tableReportSchema = {
+  type: 'object',
+  properties: {
+    data: {
+      type: 'object',
+      properties: {
+        header: {
+          type: 'array',
+        },
+        rows: {
+          type: 'array',
+          items: { type: 'array' },
+        },
+        formatters: {
+          type: 'array',
+          items: { type: 'string' },
+        },
+      },
+      required: ['header', 'rows', 'formatters'],
```

Comment:
formatters is not required

Reply:
I didn't know that haha nice! I'll remove the required

Reply:
Done here e7779b1